scala-mode.el

```diff
@@ -163,6 +163,7 @@ When started, runs `scala-mode-hook'.
         fixup-whitespace                'scala-indent:fixup-whitespace
         delete-indentation              'scala-indent:join-line
         indent-tabs-mode                nil
+        adaptive-fill-regexp            "[ \t]*\\(//+[ \t]*\\)*"
```

Comment:
you didn't add it now to the list above on line 108. I think you need to do that to be completely compliant.

However, now that I look above, we have already `adaptive-fill-function`. Did you look at the emacs manual what is the role of these with respect to each other. (Ctrl-h+f)

Reply:
Yeah, I didn't add it up there because my testing didn't reveal a need to---but I see now that it would make sense to do that and keep consistency.

I did test this with multi-line comments. They go down a slightly different codepath, and are unaffected. I will test with scala-doc comments too, but I think they will work just like multi-line comments.

This change was a result of tracing through the code quite a bit, and also of reading quite a bit of the docs, though I don't remember the exact details now of what I read in `adaptive-fill-function`.